packages/public/server/__tests-e2e__/login.ts

```diff
@@ -19,49 +19,144 @@
+    .getProfileButton(afterLoginPage)
+    .then(clickForNewPage)
 
-      const loginPage = await elements
-        .getLoginButton(firstPage)
-        .then(clickForNewPage)
+  expect(userPage).toHaveUrlPath('/user/me')
+  await expect(userPage).toMatchElement('h1', { text: 'login' })
+})
+
+describe('wrong login-data', () => {
+  let loginPage: ElementHandle
+  let afterLoginPage: ElementHandle
+
+  beforeEach(async () => (loginPage = await goto('/auth/login')))
 
-      expect(loginPage).toHaveUrlPath(pages.login.path)
+  describe('password and user is given', () => {
+    test('wrong password and correct user', async () => {
+      afterLoginPage = await typeIntoLoginFormAndWaitForNewPage({
+        password: '123',
+        page: loginPage,
+        user: 'login',
+      })
+    })
+
+    test('correct password and wrong user', async () => {
+      afterLoginPage = await typeIntoLoginFormAndWaitForNewPage({
+        password: '123456',
+        page: loginPage,
+        user: 'abc',
+      })
+    })
 
-      const { buttonLogin, inputUser, inputPassword } = pages.login.identifier
-      await getByPlaceholderText(loginPage, inputUser).then((e) => e.type(user))
-      await getByPlaceholderText(loginPage, inputPassword).then((e) =>
-        e.type(pages.login.defaultPassword)
-      )
+    afterEach(async () => {
+      expect(afterLoginPage).toHaveUrlPath(pages.login.path)
+      expect(
+        await getByText(
+          afterLoginPage,
+          'Mit dieser Kombination ist bei uns kein Benutzer registriert.'
+        )
+      ).toBeDefined()
+    })
+  })
 
-      const afterLoginPage = await getByText(loginPage, buttonLogin).then(
-        clickForNewPage
-      )
+  describe('user or password is missing', () => {
+    test('missing password and missing user', async () => {
+      await typeIntoLoginFormAndClickLoginButton({
+        page: loginPage,
+        user: '',
+        password: '',
+      })
+    })
 
-      await expect(await elements.getLogoutButton(afterLoginPage)).toBeDefined()
+    test('missing password and correct user', async () => {
+      await typeIntoLoginFormAndClickLoginButton({
+        page: loginPage,
+        user: 'login',
+        password: '',
+      })
+    })
 
-      const userPage = await elements
-        .getProfileButton(afterLoginPage)
-        .then(clickForNewPage)
+    test('correct password and missing user', async () => {
+      await typeIntoLoginFormAndClickLoginButton({
+        page: loginPage,
+        user: '',
+        password: '123456',
+      })
+    })
 
-      expect(userPage).toHaveUrlPath('/user/me')
-      await expect(userPage).toMatchElement('h1', { text: user })
+    afterEach(async () => {
+      expect(await getCurrentPage()).toHaveUrlPath('/auth/login')
     })
   })
 })
+
+async function typeIntoLoginForm(arg: {
+  page: ElementHandle
+  user: string
+  password: string
+}) {
+  const { inputUser, inputPassword } = pages.login.identifier
```

Comment:
Can you check whether we can delete `pages.login` of `_config.ts` and hard code its elements into this file?